Priority_Queue/Priority_queue.js

```diff
@@ -0,0 +1,99 @@
+class queueElement 
+{
+  constructor(element,priority)
+  {
+    this.element=element;
+    this.prioriry=priority;
+  }
+}
+
+class PriorityQueue {
+  constructor()
+  {
+    this.items=[];
+  }
+
+//to add an element to the queue
+  enqueue(element,priority)
+  {
+    var newQueueElement=new queueElement(element,priority);
+    var isPresent=false;
+    for (let i = 0; i < this.items.length; i++) { 
+		if (this.items[i].priority > newQueueElement.priority) { 
```

Comment:
I think you can do a binary search to find the optimum position to put the current element. This would cause the loop to go through complete N iterations if priority provided for the new element is least.

Reply:
Yes but the implementation will get tougher

Reply:
Please review and add it so that i can work on my next issue